src/read-file.ts

```diff
@@ -25,9 +24,6 @@ export async function readFile<T>(filePath: string): Promise<T> {
     case ".mjs":
       return importJavaScriptModule(filePath);
 
-    case ".ts":
```

Comment:
We should throw an error if they reference a .ts file.  The error message should say something like:

> Invalid file reference: "scripts/my-script.ts".  Reference the compiled JavaScript file instead of the TypeScript source file.